docs/data/introduction/support/support.md

```diff
@@ -33,3 +33,19 @@ Including your Order ID (or Support key) in the issue helps us prioritize the is
 1. MUI X Pro: maintainers give these issues more attention than the ones from the community.
 2. MUI X Premium: The same as MUI X Pro, but with higher priority.
 3. MUI X Priority Support add-on (coming soon): Provides a 24h SLA for the first answer.
+
+### Long-term support (LTS)
+
+We roll bug fixes, performance enhancements, and other improvements into new releases, however, MUI remains committed to delivering security updates and addressing regressions for the immediate predecessor of the current major version.
```

Comment:
That section sounds a bit weird. Would you mind a slight restructure?

```suggestion
While each new release of MUI includes bug fixes, performance enhancements, and other improvements, we also remain committed to delivering security updates and addressing regressions for the immediate predecessor of the current major version.
```

Reply:
```suggestion
Bug fixes, performance enhancements, and other improvements are delivered in new releases. However, MUI remains committed to delivering security updates and addressing regressions for the immediate predecessor of the current major version.
```